sktree/tree/_unsup_criterion.pyx

```diff
@@ -263,44 +302,6 @@ cdef class TwoMeans(UnsupervisedCriterion):
     pair minimizes the splitting criteria described in the following
     section
     """
-    cdef double sum_of_squares(
```

Comment:
Don't move this to base class.

Reply:
moved this to TwoMeans and subclassed FastBIC to it